src/features/bookings/components/BookingDetailsCancelButton.tsx

```diff
@@ -45,11 +51,16 @@ export const BookingDetailsCancelButton = (props: BookingDetailsCancelButtonProp
         </React.Fragment>
       )
     } else {
+      const displayMessage =
+        'Tu ne peux plus annuler ta réservation : elle devait être annulée avant le {date}'
+      const displayMessageHeader = isExBeneficiary ? 'Ton crédit est expiré.\n' : ''
```

Comment:
```javascript
        <React.Fragment>
          {!!isExBeneficiary && (
            <CancellationCaption>{t`Ton crédit est expiré.`}</CancellationCaption>
          )}
          <CancellationCaption>
            {t({
              id: 'not cancellable because expired',
              values: { date: formattedConfirmationDate },
              message:
                'Tu ne peux plus annuler ta réservation : elle devait être annulée avant le {date}',
            })}
          </CancellationCaption>
        </React.Fragment>
```

pour éviter de faire une clef de traduction qui dépende du code (et éviter le `\n` dans le message)

Reply:
je trouve ça plus lourd non ?